driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/OrientGraph.java

```diff
@@ -52,11 +62,10 @@ public static OrientGraph open(final Configuration configuration) {
     public OrientGraph(Configuration config) {
         this.url = config.getString(CONFIG_URL, "memory:test-" + Math.random());
         this.database = getDatabase(url,
-            config.getString(CONFIG_USER, "admin"),
-            config.getString(CONFIG_PASS, "admin"),
-            config.getBoolean(CONFIG_CREATE, true),
-            config.getBoolean(CONFIG_OPEN, true));
-        makeActive();
+                config.getString(CONFIG_USER, "admin"),
```

Comment:
It's a continuation of a previous line. Makes sense here. Automatically formatted